osu.Game/Graphics/UserInterface/OsuTabDropdown.cs

```diff
@@ -66,7 +66,7 @@ public OsuTabDropdownMenu()
                 Origin = Anchor.TopRight;
 
                 BackgroundColour = Color4.Black.Opacity(0.7f);
-                MaxHeight = 400;
+                MaxHeight = 200;
```

Comment:
Not super sure about changing this at a global level, but doesn't seem to be affecting anything.